storyblok/storyblok-js-client
# `INT-123` - eat/int 123 remove axios

## Pull request type

Removal of axios' dependency.

Jira Link: [INT-123](https://storyblok.atlassian.net/browse/INT-123)




## How to test this PR


This PR adds updated information to the README file, related to axios' removal.
You don't need to install axios anymore. So, you might try a brand new installation, like described on the updated README file.
You should make tests by reproducing the same behavior of the SDK with axios. Instantiate the `StoryblokClient` on a dummy application, as described on our documentation and try to make all the API requests. You should also install a `responseInterceptor` and try to remove it with the new function `ejectInterceptor`. 

## What is the new behavior?



- A new function to eject the responseInterceptor was added: `ejectInterceptor`
- The Proxy option was removed.

## Other information

Any questions, feel free to ask me.
